assignment/scripts/array-practice.js

```diff
@@ -4,77 +4,87 @@ console.log('****** Array Practice *******');
 console.log('--- 1. Practice creating arrays ---');
 
 // Example: Array of animals
-let animalArray = ['fish', 'cat', 'bird', 'dog'];
-console.log('Animals are: ', animalArray);
+// let animalArray = ['fish', 'cat', 'bird', 'dog'];
+// console.log('Animals are: ', animalArray);
 
 // 1.a. TODO: Make an array with some favorite foods
-
+let favoriteFoods = [ 'cannoli', 'italian ice', 'sfogliatella', 'rainbow cookie'];
 
 // 1.b. TODO: Log your array of foods to the console with a message, similar 
 //      to the example above
-
+console.log('My favorite Italian desserts are:', favoriteFoods);
 
 
 // 2. Array.length - tells you how many items are in the array
 console.log('--- 2. Length of an array ---');
 
 // Example: How many animals are in the array? 
-console.log('Number of animals:', animalArray.length);
+// console.log('Number of animals:', animalArray.length);
 
 // 2.a. TODO: Log to the console the number of foods in your array
-
+console.log('Number of favorite Italian desserts:', favoriteFoods.length);
 
 // 3. Accessing array items
 console.log('--- 3. Accessing items in an array ---');
 
 // Example: Log the first animal from the array using it's array index
-console.log('First animal is', animalArray[0])
-
-// 3.a. TODO: Log the second animal in the array 
-
+// console.log('First animal is', animalArray[0])
 
-// 3.b. TODO: Log the last animal in the array using it's array index 
+// 3.a. TODO: Log the second food in the array 
+console.log('Second dessert is:', favoriteFoods [1]);
 
+// 3.b. TODO: Log the last food in the array using it's array index 
+console.log('Last dessert is:', 'rainbow cookie');
```

Comment:
Not quite. The prompt said to use the last items's "array index", but you just copied and pasted the string.